### DISCUSSION TITLE
Are there any new plans to adapt NextJS v13 app directory route handlers?

### DISCUSSION THREAD

**Discussion by Sonequa** [nextauthjs/next-auth#7193] [Ideas]
Since NextJS v13 has been released for several months, there's still no support in next-auth for the [route handlers](https://beta.nextjs.org/docs/routing/route-handlers#request-body) in the app directory. Knowing that the app directory itself is still in beta so it's okay not to keep up with it so far. But still feel sad every time I get stuck while trying to migrate my project from the page directory to the app directory because of the mismatch of next-auth. Just want to know if is there any plan for it or are you guys already in progress. Best wishes at last

**Comment by philwolstenholme** [comment]
Released four hours ago as part of 4.22.0! https://twitter.com/nextauthjs/status/1645004172725551104
